migrations/migrations/20181023175544-users-model.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('users', {
+      id: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        unique: true
+      },
+      name: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      surname: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      email: {
+        type: Sequelize.STRING,
+        allowNull: false,
+        unique: true
+      },
+      password: {
+        type: Sequelize.STRING,
+        allowNull: false
+      }
```

Comment:
You also need to add some field for timestamps like `created_at` and `updated_at` or add to the model the option of `timestamps: false` so the orm knows that you don't have those columns in the table.

Reply:
I actually fixed that on the userCreate pr, should I add those fixes to this branch as well?

Reply:
let's leave it there